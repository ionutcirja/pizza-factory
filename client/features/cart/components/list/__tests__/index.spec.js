import React from 'react';
import { shallow } from 'enzyme';
import { List } from '..';

describe('List component', () => {
  const propsToRender = {
    list: ['1', '2', '3'],
  };
  
  describe('render', () => {
    it('should render a list of items', () => {
      const wrapper = shallow(<List {...propsToRender} />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
