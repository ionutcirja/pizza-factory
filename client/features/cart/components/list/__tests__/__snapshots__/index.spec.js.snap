// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List component render should render a list of items 1`] = `
<CartList>
  <Connect(WithTheme(Item))
    id="1"
    key="1"
  />
  <Connect(WithTheme(Item))
    id="2"
    key="2"
  />
  <Connect(WithTheme(Item))
    id="3"
    key="3"
  />
</CartList>
`;
